Khan/perseus
# [Interactive Graph] Open and Closing logic for unlimited polygon.

## Summary:
Adding opening and closing behavior to unlimited polygon including:
- New buttons for graph controls.
- Reducing redundancy between polygon and unlimited polygon behavior.
- And more robust testing for unlimited graph controls.

Issue: LEMS-2570

## Test plan:
- Run `yarn dev` and enable `Unlimited Polygon` option in the Mafs dropdown. Or http://localhost:6006/?path=/story/perseus-widgets-interactive-graph--unlimited-polygon-with-mafs.
- Go the the exercise and build polygons with mouse or keyboard.
- Use the `Close polygon` button the close the polygon.
- The expectation is now the polygon should look and behave like other Mafs polygon graphs.
- Use the `Open polygon` button again to add or remove points.
